tests/conftest.py

```diff
@@ -323,11 +325,18 @@ def init_config():
             ("configure_scenario_from_tasks", ScenarioConfig._configure_from_tasks),
         ],
     )
+    _inject_section(
+        VersionMigrationConfig,
+        "version_migrations",
+        VersionMigrationConfig.default_config(),
+        [("set_version_migration", VersionMigrationConfig._set)],
+    )
```

Comment:
What do you think about the method name `Config.set_version_migration()`?

Reply:
it's not really "add", but "set" functions for the migration. How about `Config.set_migration_functions()`?

Reply:
Oh, you are right. That's because I misunderstood the VersionMigrationConfig section. I though it was a unique section.
If we implement it as a Section, I agree that `set_migration_function` is better.
If we implement it as a UniqueSection, I prefer `add_migration_function`.